src/environment.cpp

```diff
@@ -256,6 +260,13 @@ void Environment::stepTimeOfDay(float dtime)
 	}
 }
 
+u32 Environment::getDayCount()
+{
+	MutexAutoLock lock(this->m_time_lock);
```

Comment:
Check with the latest push please, I wasn't entirely sure what atomic construct was used.